hw/block/nvme.c

```diff
@@ -49,6 +49,44 @@
             " in %s: " fmt "\n", __func__, ## __VA_ARGS__); \
     } while (0)
 
+
```

Comment:
@yuval please add sing off to commit message, I think we should review the commit guidelines of Qemu project